public/app/features/logs/components/LogDetailsRow.tsx

```diff
@@ -129,6 +130,7 @@ class UnThemedLogDetailsRow extends PureComponent<Props, State> {
       fieldType: isLabel ? 'label' : 'detectedField',
       type: showFieldsStats ? 'close' : 'open',
       logRowUid: row.uid,
+      app: app,
```

Comment:
Just fyi: you can shorthand property names if the key and the variable named the same. So you could just write it like this:

```suggestion
      app,
```